src/services/updateCameraConfig.ts

```diff
@@ -19,22 +18,19 @@ export const UpdateCameraConfigService: IService = {
         {
           encoding: 'utf-8',
         },
-        () => {
-          if (ifTimeSet()) {
-            exec(CMD.STOP_CAMERA, () => {
-              exec(CMD.START_CAMERA, () => {
-                console.log('Successfully restarted the camera');
-              });
-            });
-          } else {
-            // do not restart the camera for image rotation
-            // it will be restarted on 3d lock anyways
-          }
-        },
+        () => restartCamera(),
       );
     } catch (e: unknown) {
       console.log('Camera Config service failed with error', e);
     }
   },
   delay: 200,
 };
+
+const restartCamera = () => {
+  exec(CMD.STOP_CAMERA, () => {
+    exec(CMD.START_CAMERA, () => {
```

Comment:
@h-tohkubo 
can we add 1-2 sec sleep before executing start?
I noticed how scheduling commands like this one RIGHT after another sometimes brings issues on Pi
like, system isn't fully got that process stopped, and we're kicking start

Reply:
Yup! Added sleep.